### ISSUE TITLE
Johktari BM's Wounded Prey should work on Guardain Sslak's

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#290]
The Gamehost is responsible for setting up the Domination scenario this means the Sslak's are "Friendly" to her.  This is something that we need to figure out we need to stop depending on OCTGN control to represent a Friendly creature.

**Comment by ACG8** [comment]
We could use card properties to determine which player is the controller and which player is the owner of a card (store player ids as ints). Then we just make sure scenario-controlled cards are owned and controlled by neither player.

**Comment by sIKE23** [comment]
I was thinking something close, we could have a hidden property "Friendly"
that is set to True by default, and then we could change it to False if it
is Scenario, if it is a Team Game, set it to the Friendly Team value that
we decide on later. Your new script tool should make this easy. I think my
idea might be a bit less flexible than yours.

//FC

**Comment by ACG8** [comment]
Well, the property can't just say whether or not a card is friendly; it has to specify friendly relative to which player (hence the player ids).

**Comment by sIKE23** [comment]
True! For Co-Op Team play we could replace the player ids with team ids.

//FC

**Comment by ACG8** [comment]
If teams are defined by lists of player ids (which I think is probably the easiest way to do it), then we still only need to associate cards with player ids, and we can calculate friendliness by whether a card is controlled by a player contained in the same team as you (the player trying to find out if a card is friendly). I think it would be best to just stick with the basic definition of "friendly if controlled by a friendly player", and handle control ourselves (as opposed to our previous use of OCTGN's definition of card control).
